models/blog-category.ini

```diff
@@ -0,0 +1,11 @@
+[model]
+name = Blog Category
+label = {{ this.name }}
+hidden = yes
+
+[children]
+replaced_with = site.query('archives/' + this.parent.path.split('/')[-1]).filter(F.categories.contains(this)) 
```

Comment:
All blogs will not be under `archives`.

Reply:
sorry @kgodey, I didn't not got the email notification for this. 
Okay, I will look into it and try to structure it differently.